webapp/src/ts/services/rules-engine.service.ts

```diff
@@ -102,15 +102,17 @@ export class RulesEngineService implements OnDestroy {
           .all([
             this.settingsService.get(),
             this.userContactService.get(),
-            this.userSettingsService.get()
+            this.userSettingsService.get(),
+            this.chtScriptApiService.getApi()
           ])
-          .then(([settingsDoc, userContactDoc, userSettingsDoc]) => {
+          .then(([settingsDoc, userContactDoc, userSettingsDoc, chtScriptApi]) => {
             const rulesSettings = this.getRulesSettings(
```

Comment:
The arguments passed to `getRulesSettings` seem to be identical to the ones passed to `monitorChanges` and `rulesConfigChange`. Do you think it would be an idea to create a function that generates an object with this properties and have just one argument for each of these functions?

Reply:
Done!